vendor/bat-native-ads/src/bat/ads/internal/conversions/verifiable_conversion_envelope_unittest_util.cc

```diff
@@ -103,7 +103,7 @@ absl::optional<std::string> OpenEnvelope(
   const std::vector<uint8_t> plaintext =
       Decrypt(ciphertext, nonce, ephemeral_public_key, advertiser_secret_key);
 
-  return (const char*)&plaintext.front();
```

Comment:
this won't lead to dangling pointer as plaintext content will be copied to temporary optional<std::string>

Reply:
Agreed to merge to fix cppclean warning and to make code clearer to read